packages/block-library/src/image/index.php

```diff
@@ -230,6 +230,8 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 	// contain a caption, and we don't want to trigger the lightbox when the
 	// caption is clicked.
 	$p->set_attribute( 'data-wp-on-async--click', 'actions.showLightbox' );
+	$p->set_attribute( 'data-wp-class--hide', 'state.isContentHidden' );
+	$p->set_attribute( 'data-wp-class--show', 'state.isContentVisible' );
```

Comment:
I assume we can't use `hidden` or `!state.isContentHidden` instead of having different properties because of the animations. Is that right?

Reply:
What would `hidden` be in this case? A class, or a directive of some kind?

As far as `!state.isContentHidden` goes, yes doing that causes the animation to trigger on page load, which we don't want. I suppose we could introduce some intermediary logic to ensure the animation only triggers after initialization, but it seems to me that would complicate the code for little gain. What do you think?

Reply:
It works for me 🙂 I was just trying to understand it better.